### ISSUE TITLE
IYOKAN - Fn + backspace = deleting all text

### ISSUE THREAD

**Issue by tomasSK** [LegacyXperia/local_manifests#407]
CM VERSION: CM11
RELEASE DATE: 20140113 

Combination of Fn key + backspace key on QWERTY keyboard at same time causes deleting all text. Not just one letter from text. I think this issue is presented in all LX builds (after CM10). 

DEVICE: IYOKAN

**Comment by mikeNG** [comment]
keyboard layout is the same as cm10 in 10.1/10.2/11.0

**Comment by tomasSK** [comment]
Ok, but this key kombination is driving me crazy. Is it possible to disable it? Thank you Mike.

**Comment by mochidawn** [comment]
Pressing Fn + backspace also causes deleting all text in Sony's official ROM, and indeed it is bothering.

**Comment by mikeNG** [comment]
Unfortunately I don't have a device with hardware keyboard to make some tests, but this is the expected behaviour (since it's the same in stock roms too)

I don't think there is an easy way to disable that combination but I'll check it when I find some time.

**Comment by tomasSK** [comment]
I don't think there was the same behavior on stock rom because it never
happened to me before. ;-)

2014-02-03 Michael Bestas notifications@github.com:

**Comment by A-n-D** [comment]
no, this is not the same in stock "4.0.4 ICS Build 4.1.B.0.587" and "2.3.4 GB Build 4.0.2.A.0.62"
i tested in both ics and gb stock roms, it just simply delete one character normally. 

but in LXP roms this combination delete all text and IT'S NOT A FEATURE..
while chating it really makes me insane cause then i need to write those all long text again.

did u find any workaround to off this manually?,
thank u mike.

**Comment by mikeNG** [comment]
Please provide the keycodes of backspace & FN keys (you can find them through cwm key test)

**Comment by A-n-D** [comment]
function Key: 56
back space Key: 14

**Comment by A-n-D** [comment]
i'm on cm-10.1 latest nightly,  and the phone is xperia pro mk16i.

**Comment by Itchythefirst** [comment]
Hi, I can confirm, that this is a big "WTF moment" when I delete the whole text :-(( It is not like it used to be on stock ROM. Please if You would be so kind and fix it in some future build, thank You.

**Comment by Itchythefirst** [comment]
Would you please take a look at this "feature"? The current behavior is really insane :-( Thank you very  much.

**Comment by fedcas** [comment]
i confirm it's really annoying :(

**Comment by mikeNG** [comment]
I was donated an xperia pro so I might be able to take a look at this finally

**Comment by fedcas** [comment]
awesome!

**Comment by mikeNG** [comment]
This is actually working as intented https://github.com/CyanogenMod/android_frameworks_base/blob/cm-12.1/core/java/android/text/method/BaseKeyListener.java#L77
